### PR TITLE
Added an explanation

### PR DESCRIPTION
**Pull request by SteveALee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +21,7 @@ funcpack pack ./
 
 You can then test locally using the CLI tool: `func run <myfunc>`
 
-When uploading your files, you need to include your `package.json`, but you don't need your `node_modules` directory.
+When uploading your files, you need to include the single `.funcpack` directory (in the Functions App root), but you don't need your `node_modules` directory for each Function.
```
Comment (christopheranderson) [inline README.md]: You shouldn't include a node_modules folder for each Functions. Ideally, you have one set of dependencies. That helps reduce the size of your final pack.

Comment (SteveALee) [inline README.md]: I totally agree - at least you should pull the common modules up to the function app. However none of the examples or docs demonstrate that. I'll chane the wording to cover all cases

Why did you mention package.json?

Comment (SteveALee) [inline README.md]: PS did you look at tree shaking? It's still a bit experimental in webpack 2 but could help. I guess how much effort you put in depends on how long before a file systems fix lands :)

**Comment by SteveALee** [comment]
@christopheranderson is that OK now?

**Pull request merged**